Android CI is failing

Our CI is failing on Android with:

```
Build WGPU example for Android
  Warning: You use environment variable ANDROID_HOME that is deprecated. Please, remove it and use ANDROID_SDK_ROOT instead. Now ANDROID_HOME is used
  Error: Android NDK is not found. Please set the path to the Android NDK with $ANDROID_NDK_ROOT environment variable.
  Error: Process completed with exit code 1.
```

Example: https://github.com/EmbarkStudios/rust-gpu/runs/7635577970?check_suite_focus=true

May be related to the following GitHub Action ubuntu image updates:
- https://github.com/actions/virtual-environments/issues/5930
- https://github.com/actions/virtual-environments/issues/5879

--- a/.github/workflows/ci.yaml
+++ b/.github/workflows/ci.yaml
@@ -94,7 +94,8 @@ jobs:
           echo "::endgroup::"
 
           echo "::group::Install cargo-apk"
-          cargo install cargo-apk
+          echo "FIXME(eddyb) working around libc 0.2.127 breakage by using stable Rust!"
+          cargo +stable install cargo-apk
           echo "::endgroup::"
 
           echo "::group::Add aarch64-linux-android target"

--- a/examples/runners/wgpu/build.rs
+++ b/examples/runners/wgpu/build.rs
@@ -35,6 +35,7 @@ fn main() -> Result<(), Box<dyn Error>> {
             "--target-dir",
         ])
         .arg(dir)
+        .env_remove("CARGO_ENCODED_RUSTFLAGS")
         .stderr(std::process::Stdio::inherit())
         .stdout(std::process::Stdio::inherit())
         .status()?;

